packages/@monocdk-experiment/rewrite-imports/package.json

```diff
@@ -43,5 +49,8 @@
   "stability": "experimental",
   "engines": {
     "node": ">= 10.3.0"
+  },
+  "dependencies": {
+    "glob": "^7.1.6"
   }
```

Comment:
Nitpicking, but could we move this out above `devDependencies`? I like to keep "dependencies" blocks close together.

Reply:
done sir